mbuild/packing.py

```diff
@@ -37,7 +39,8 @@
 """
 
 
-def fill_box(compound, n_compounds=None, box=None, density=None, overlap=0.2, seed=12345):
+def fill_box(compound, n_compounds=None, box=None, aspect_ratio=None,
+        density=None, overlap=0.2, seed=12345, write_tempfile=False):
```

Comment:
Drive by comment: let's append the new args to the end just to avoid breaking any code that called this with positional args.